scripts/fortran_tools/offline_check_fortran_vs_metadata.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Can you make this file executable so a user does not have to type `python` or `python3` first?

Reply:
Yes! I think I have successfully updated the git index to make it executable - it appears as such when i do a clean checkout